README.md

```diff
@@ -113,6 +147,22 @@ async main() {
 }
 ```
 
+Disabling retry behavior will not prevent the usage of other options set on the `options` object.
+This example showcases a disabled retry behavior, but custom headers will still be set for the request:
```

Comment:
I think this is overkill - this should be clear since we say standard options are supported.

Reply:
Changed.